ampd/src/tm_client.rs

```diff
@@ -24,10 +24,7 @@ impl TmClient for HttpClient {
     async fn latest_block(&self) -> Result<BlockResponse, Error> {
         future::with_retry(
             || Client::latest_block(self),
-            RetryPolicy::RepeatConstant {
-                sleep: Duration::from_secs(1),
-                max_attempts: 15,
-            },
+            RetryPolicy::repeat_constant(Duration::from_secs(1), 15),
```

Comment:
why are these values hardcoded?

Reply:
ok let's leave them for now. I've made a note so we can come back and clean it up later